cli/cmd/plugin/ui/web/tanzu-ui/src/assets/contextualHelpDocs/data.json

```diff
@@ -1 +1,54 @@
kage list in your terminal to view avaliable packages on an unmanaged cluster, or workload cluster.</p>\n\n<a style=\"color: white\" href=\"https://tanzucommunityedition.io/docs/main/architecture/#tanzu-clusters\" target=\"_blank\"\n    >Learn more about managing Tanzu Packages.</a>\n"},{"topicTitle":"Troubleshooting cluster creation","topicIds":["tce-getting-started"],"htmlContent":"\n\n<p>If your bootstrap cluster fails to pivot to a management cluster by hanging or returning an error similar to the following:</p>\n\n<code\n    >Error: unable to set up management cluster: unable to wait for cluster and get the cluster kubeconfig: error waiting for cluster to be\n    provisioned (this may take a few minutes): cluster creation failed, reason:'NatGatewaysReconciliationFailed', message:'3 of 8 completed'\n</code>\n\n<p>Access the pod log for your respective provider to identify the problem.</p>\n\n<p>\n    ## For AWS <br />\n    <code>\n        docker exec &lt;CONTAINER_ID&gt; kubectl logs --namespace capa-system --selector\n        cluster.x-k8s.io/provider=infrastructure-aws,control-plane=controller-manager -c manager --kubeconfig /etc/kubernetes/admin.conf\n    </code>\n</p>\n<p>\n    ## For Azure <br />\n    <code>\n        docker exec &lt;CONTAINER_ID&gt; kubectl logs --namespace capz-system --selector\n        cluster.x-k8s.io/provider=infrastructure-azure,control-plane=controller-manager -c manager --kubeconfig /etc/kubernetes/admin.conf\n    </code>\n</p>\n<p>\n    ## For Docker <br />\n    <code>\n        docker exec &lt;CONTAINER_ID&gt; kubectl logs --namespace capd-system --selector\n        cluster.x-k8s.io/provider=infrastructure-docker,control-plane=controller-manager -c manager --kubeconfig /etc/kubernetes/admin.conf\n    </code>\n</p>\n<p>\n    ## For vSphere <br />\n    <code>\n        docker exec &lt;CONTAINER_ID&gt; kubectl logs --namespace capv-system --selector\n        cluster.x-k8s.io/provider=infrastructure-vsphere,control-plane=controller-manager -c manager --kubeconfig /etc/kubernetes/admin.conf\n    </code>\n</p>\n"},{"topicTitle":"Unmanaged Clusters","topicIds":["tce-welcome","tce-getting-started"],"htmlContent":"\n\n<p>\n    An unmanaged cluster offers a single node, local workstation cluster suitable for a development/test environment. It requires minimal\n    local resources and is fast to deploy. It provides support for running multiple clusters. The default Tanzu Community Edition package\n    repository is automatically installed when you deploy an unmanaged cluster.\n</p>\n"},{"topicTitle":"Workload Clusters","topicIds":["tce-getting-started"],"htmlContent":"\n\n<p>\n    After you deploy the management cluster, you can deploy a workload cluster. The workload cluster is deployed by the management cluster.\n    The workload cluster is used to run your application workloads. The workload cluster is deployed using the Tanzu CLI.\n</p>\n\n<p>A management cluster is required in order to view workload clusters.</p>\n"}]}
```

Comment:
Are you working on the help docs? If not, probably best to revert these changes.

Reply:
That was prettier I am not sure how to stop this from happening